src/Server/config.ts

```diff
@@ -107,6 +107,7 @@ export const SEGMENT_WRITE_KEY_SERVER: any = null
 export const SEGMENT_WRITE_KEY: any = null
 export const SENTRY_PRIVATE_DSN: any = null
 export const SENTRY_PUBLIC_DSN: any = null
+export const SENTRY_TRACING_ENABLED: any = false
```

Comment:
nitpick: boolean?

Reply:
It's coming from an environment variable so you can set whatever. Something does apparently coerce though.